### PR TITLE
Errors display when password reset token is blank

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/main/java/page_object/BasePage.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
     private final By btnChangePasswordTab = By.cssSelector("a[href*='ChangePassword']>span");
     private final By btnMyTicketTab = By.cssSelector("a[href*='ManageTicket']>span");
     private final By lblHeader = By.cssSelector("div#content > h1[align='center']");
+    private final By likForgotPassword = By.cssSelector("#page > #content > ul > li > a[href*='ForgotPassword']");
```
Comment (anhlvt) [inline src/main/java/page_object/BasePage.java:19]: Could we make the css more shortly?

Comment (Baonganbui123) [reply src/main/java/page_object/BasePage.java]: I fixed it, please check again!

File: src/main/java/page_object/BasePage.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
     private final By btnChangePasswordTab = By.cssSelector("a[href*='ChangePassword']>span");
     private final By btnMyTicketTab = By.cssSelector("a[href*='ManageTicket']>span");
     private final By lblHeader = By.cssSelector("div#content > h1[align='center']");
+    private final By likForgotPassword = By.cssSelector("a[href*='ForgotPassword']");
```
Comment (MinhQuan119) [inline src/main/java/page_object/BasePage.java:19]: `lnk` stands for `link`
Most of the case, we often omit vowel when we abbreviate

Comment (Baonganbui123) [reply src/main/java/page_object/BasePage.java]: Please check again!

**Review by MinhQuan119 — APPROVED** [review]

**Review by anhlvt — APPROVED** [review]

**Pull request merged**